test/Microsoft.TemplateEngine.TestHelper/EnvironmentSettingsHelper.cs

```diff
@@ -38,16 +38,6 @@ public EnvironmentSettingsHelper(IMessageSink messageSink)
             {
                 locale = "en-US";
             }
-            var builtIns = new List<(Type, IIdentifiedComponent)>();
```

Comment:
what impact it has on tests relying on this functionality?

Reply:
This is a dead (and duplicate) code - as it's not passed to `TestHost` and `TestHost` is instantiation builtins by itself: https://github.com/dotnet/templating/blob/main/test/Microsoft.TemplateEngine.TestHelper/TestHost.cs#L38-L47